.github/workflows/CI.yml

```diff
@@ -1,18 +1,13 @@
 name: CI
 
 on:
-  push:
-    branches:
-      - main
-  pull_request_target:
-    branches:
-      - main
+  pull_request:
 
 jobs:
   lint-and-test:
     strategy:
       matrix:
-        node: ["14.x", "15.x"]
+        node: ["14.16.0", "15.x"]
```

Comment:
14.16.1 is out : maybe use 14.x or 14.16.x or 14.16.1 instead of 14.16.0 ?
(to be changed elsewhere also)